package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wonderlandengine/react-ui",
-  "version": "0.1.0-dev.7",
+    "version": "0.1.0-dev.9",
```

Comment:
Formatting seems off here. Maybe format the file with the prettier-config package?

Reply:
I formatted the file correctly.